src/components/connections/EditConnectionView.ts

```diff
@@ -51,13 +63,15 @@ export class EditConnectionView extends View {
           path.join(this.context.extensionPath, "templates", "editConn.js")
         )
       ),
+      isV1: conn?.version === InfluxConnectionVersion.V1,
       title: title,
-      connID: conn !== undefined ? conn.id : "",
-      connName: conn !== undefined ? conn.name : "",
+      connID: conn?.id,
+      connName: conn?.name,
+      defaultHostNPortV1: defaultHostNPortV1,
       defaultHostNPort: defaultHostNPort,
-      connToken: conn !== undefined ? conn.token : "",
-      connOrg: conn !== undefined ? conn.org : "",
-      isDefault: conn !== undefined && conn.isDefault ? "checked" : ""
+      connToken: conn?.token,
```

Comment:
might be better to make these properties of connection required and set defaults